plugins/modules/lvg.py

```diff
@@ -146,6 +146,13 @@
     state: active
     vg: vg.services
 
+- name: Add new PVs to volume group without removing existing ones
+  community.general.lvg:
+    vg: vg.services
+    pvs: /dev/sdb1,/dev/sdc1
```

Comment:
For the next time: it's better to use YAML lists instead of comma-separated strings for lists.